Synchronize MODULE_LICENSE() with the stated license header.

[2009-07-22, 19:03:11] < mcgrof> pkgadd: I asked hch and he's OK with using MODULE_LICENSE("Dual GPL/BSD") and changing hte header on the probe.c file to ISC

Not-Signed-off-by: Stefan Lippers-Hollmann <s.l-h@gmx.de>

--- a/drivers/staging/ar5523/ar5523.c
+++ b/drivers/staging/ar5523/ar5523.c
@@ -1661,7 +1661,7 @@ static void __exit ar5523_exit(void)
 	usb_deregister(&ar5523_driver);
 }
 
-MODULE_LICENSE("GPL");
+MODULE_LICENSE("Dual BSD/GPL");
 
 module_init(ar5523_init);
 module_exit(ar5523_exit);
